src/widgets/stats/__tests__/stats-test.ts

```diff
@@ -0,0 +1,164 @@
+import { render } from 'preact';
+import stats from '../stats';
+import { castToJestMock } from '../../../../test/utils/castToJestMock';
+
+jest.mock('preact', () => {
+  const module = jest.requireActual('preact');
+
+  return {
+    ...module,
+    render: jest.fn(),
+  };
+});
```

Comment:
```suggestion
import { render as preactRender } from 'preact';
import stats from '../stats';
import { castToJestMock } from '../../../../test/utils/castToJestMock';

const render = castToJestMock(preactRender);
jest.mock('preact', () => {
  const module = jest.requireActual('preact');

  module.render = jest.fn();

  return module;
});
```

(then remove the calls to castToJestMock in the actual tests)

This is more consistent with the other tests